README.md

```diff
@@ -30,8 +30,8 @@ Instalation
 =
 Dependencies will be automatically installed.
 
-    git clone https://github.com/LionSec/xerosploit
-    cd xerosploit && sudo python install.py
+    git clone https://github.com/shivanshjoshi28/xerosploit
```

Comment:
I believe this line should be kept as `git clone https://github.com/LionSec/xerosploit`